crypto/keys/bls12_381/key_cgo.go

```diff
@@ -151,11 +145,6 @@ func (pubKey PubKey) VerifySignature(msg, sig []byte) bool {
 		return false
 	}
 
-	if len(msg) > bls12381.MaxMsgLen {
-		hash := sha256.Sum256(msg)
-		msg = hash[:]
-	}
-
```

Comment:
some leftover tech debt here? doesn't seem related to SS.

Reply:
yea ill make a pr to main with these changes